NoSuchMethodError trying to use newer version of Ebean

Hi,

We are currently using Ebean 13.22.0 and we would like to use the new 13.26.1 version.
However, since Ebean 13.24.0, the `DatabaseConfig` do not return `void` anymore, resulting in an exception because signature mismatch.

```
play.api.UnexpectedException: Unexpected exception[NoSuchMethodError: 'void io.ebean.config.DatabaseConfig.setName(java.lang.String)']
        at play.core.server.DevServerStart$DevServerApplicationProvider$1.reload(DevServerStart.scala:257)
        at play.core.server.DevServerStart$DevServerApplicationProvider$1.get(DevServerStart.scala:190)
        at play.core.server.PekkoHttpServer.handleRequest(PekkoHttpServer.scala:320)
        at play.core.server.PekkoHttpServer.$anonfun$createServerBinding$1(PekkoHttpServer.scala:224)
        at org.apache.pekko.stream.impl.fusing.MapAsync$$anon$30.onPush(Ops.scala:1322)
        at org.apache.pekko.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:555)
        at org.apache.pekko.stream.impl.fusing.GraphInterpreter.processEvent(GraphInterpreter.scala:506)
        at org.apache.pekko.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:400)
        at org.apache.pekko.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:662)
        at org.apache.pekko.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:532)
        at org.apache.pekko.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:637)
        at org.apache.pekko.stream.impl.fusing.ActorGraphInterpreter.org$apache$pekko$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:813)
        at org.apache.pekko.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:831)
        at org.apache.pekko.actor.Actor.aroundReceive(Actor.scala:547)
        at org.apache.pekko.actor.Actor.aroundReceive$(Actor.scala:545)
        at org.apache.pekko.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:729)
        at org.apache.pekko.actor.ActorCell.receiveMessage(ActorCell.scala:590)
        at org.apache.pekko.actor.ActorCell.invoke(ActorCell.scala:557)
        at org.apache.pekko.dispatch.Mailbox.processMailbox(Mailbox.scala:280)
        at org.apache.pekko.dispatch.Mailbox.run(Mailbox.scala:241)
        at org.apache.pekko.dispatch.Mailbox.exec(Mailbox.scala:253)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
Caused by: java.lang.NoSuchMethodError: 'void io.ebean.config.DatabaseConfig.setName(java.lang.String)'
        at play.db.ebean.DefaultEbeanConfig$EbeanConfigParser.parse(DefaultEbeanConfig.java:106)
        at play.db.ebean.DefaultEbeanConfig$EbeanConfigParser.get(DefaultEbeanConfig.java:82)
        at play.db.ebean.DefaultEbeanConfig$EbeanConfigParser.get(DefaultEbeanConfig.java:62)
        at com.google.inject.internal.ProviderInternalFactory.provision(ProviderInternalFactory.java:86)
        at com.google.inject.internal.BoundProviderFactory.provision(BoundProviderFactory.java:72)
        at com.google.inject.internal.ProviderInternalFactory.circularGet(ProviderInternalFactory.java:60)
        at com.google.inject.internal.BoundProviderFactory.get(BoundProviderFactory.java:59)
        at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
        at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:169)
        at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:45)
        at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:40)
        at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:60)
        at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
        at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
        at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:300)
        at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
        at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:169)
        at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:45)
        at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:40)
        at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:60)
        at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
        at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
        at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:300)
        at com.google.inject.internal.BoundProviderFactory.get(BoundProviderFactory.java:58)
        at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
        at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:169)
        at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:45)
        at com.google.inject.internal.InternalInjectorCreator.loadEagerSingletons(InternalInjectorCreator.java:213)
        at com.google.inject.internal.InternalInjectorCreator.injectDynamically(InternalInjectorCreator.java:186)
        at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:113)
        at com.google.inject.Guice.createInjector(Guice.java:87)
        at com.google.inject.Guice.createInjector(Guice.java:78)
        at play.api.inject.guice.GuiceBuilder.injector(GuiceInjectorBuilder.scala:200)
        at play.api.inject.guice.GuiceApplicationBuilder.build(GuiceApplicationBuilder.scala:176)
        at play.api.inject.guice.GuiceApplicationLoader.load(GuiceApplicationLoader.scala:21)
        at play.core.server.DevServerStart$DevServerApplicationProvider$1.$anonfun$reload$2(DevServerStart.scala:233)
        at play.utils.Threads$.withContextClassLoader(Threads.scala:22)
        at play.core.server.DevServerStart$DevServerApplicationProvider$1.reload(DevServerStart.scala:225)
        ... 25 common frames omitted
```

Can we bump Ebean version on Play-Ebean to solve this issue ?

Yours faithfully,
LCDP

--- a/project/Dependencies.scala
+++ b/project/Dependencies.scala
@@ -16,14 +16,13 @@ object Dependencies {
 
   object Versions {
     val play: String   = "3.0.1"
-    val ebean          = "13.23.0"
-    val ebeanJakarta   = s"$ebean-jakarta"
+    val ebean          = "15.0.1"
     val typesafeConfig = "1.4.3"
   }
 
   val ebean = libraryDependencies ++= Seq(
-    "io.ebean"           % "ebean"                % Versions.ebeanJakarta,
-    "io.ebean"           % "ebean-ddl-generator"  % Versions.ebeanJakarta,
+    "io.ebean"           % "ebean"                % Versions.ebean,
+    "io.ebean"           % "ebean-ddl-generator"  % Versions.ebean,
     "io.ebean"           % "ebean-agent"          % Versions.ebean,
     "org.playframework" %% "play-java-jdbc"       % Versions.play,
     "org.playframework" %% "play-jdbc-evolutions" % Versions.play,
@@ -36,7 +35,7 @@ object Dependencies {
   )
 
   val plugin = libraryDependencies ++= Seq(
-    "io.ebean"     % "ebean"       % Versions.ebeanJakarta,
+    "io.ebean"     % "ebean"       % Versions.ebean,
     "io.ebean"     % "ebean-agent" % Versions.ebean,
     "com.typesafe" % "config"      % Versions.typesafeConfig,
   )

--- a/project/Dependencies.scala
+++ b/project/Dependencies.scala
@@ -21,9 +21,9 @@ object Dependencies {
   }
 
   val ebean = libraryDependencies ++= Seq(
-    "io.ebean"           % "ebean"                % Versions.ebean,
-    "io.ebean"           % "ebean-ddl-generator"  % Versions.ebean,
-    "io.ebean"           % "ebean-agent"          % Versions.ebean,
+    ("io.ebean" % "ebean"               % Versions.ebean).excludeAll(ExclusionRule("com.fasterxml.jackson.core")),
+    "io.ebean"  % "ebean-ddl-generator" % Versions.ebean,
+    "io.ebean"  % "ebean-agent"         % Versions.ebean,
     "org.playframework" %% "play-java-jdbc"       % Versions.play,
     "org.playframework" %% "play-jdbc-evolutions" % Versions.play,
     "org.playframework" %% "play-guice"           % Versions.play % Test,

